public-ui/src/app/data-browser/views/survey-view/survey-react-view.component.tsx

```diff
@@ -163,13 +164,9 @@ div {
 }
 `;
 
-interface Props {
-    domainId: string;
-    searchWord: string;
-}
-
 interface State {
     survey: any;
+    surveyId: string;
```

Comment:
Similar to the ehr view, we should drop this from state and use the `id` in `urlParamsStore` when needed.